src/test/java/jewellery/inventory/helper/OrganizationTestHelper.java

```diff
@@ -25,6 +26,29 @@ public static Organization getTestOrganization() {
     return organization;
   }
 
+  public static Organization getTestOrganizationWithUser(User user) {
+    Organization organization = new Organization();
+    organization.setId(UUID.randomUUID());
+    organization.setName(ORGANIZATION_NAME);
+    organization.setAddress(ORGANIZATION_ADDRESS);
+    organization.setNote(ORGANIZATION_NOTE);
```

Comment:
This is just copied from the getTestOrganization() method. We can reuse it here like:
```suggestion
    Organization organization = getTestOrganization()
```